### PR TITLE
fix an issue in user-guide/report-billing/index

### LINKED ISSUE #4893: 【运营管理】的几个问题
https://docs.daocloud.io/ghippo/user-guide/report-billing/

1、没有GPU相关描述
image

2、文档站上标题使用的是“计费计量”， 内容使用的是 “计量计费” 和  DCE5 UI上使用的一致

image

### PR DESCRIPTION
**Pull request by WANG0608GitHub**
Fix #4893

### CODE REVIEW

**Comment by WANG0608GitHub** [comment]
see preview：

File: docs/zh/docs/ghippo/user-guide/report-billing/index.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ hide:
 
 # 运营管理
 
-运营管理通过可视化的方式，为您展示平台上统计时间范围内集群、节点、命名空间、容器组、工作空间等维度的 CPU /内存/存储的使用总量和使用率等信息。以及通过使用量、使用时间及单价等信息，自动计算出的平台消费信息。该模块默认开启所有报表统计，同时也支持平台管理员对单个报表进行手动开启或关闭，开启/关闭后将在最长 20 分钟内，平台开始/停止采集报表数据，往期已采集到的数据还将正常展示。运营管理数据最多可在平台上保留 365 天，超过保留时间的统计数据将被自动删除。您也可以通过 CSV 或 Excel 方式下载报表后进行进一步的统计和分析。
+运营管理通过可视化的方式，为您展示平台上统计时间范围内集群、节点、命名空间、容器组、工作空间等维度的 CPU /内存/存储/ GPU 的使用总量和使用率等信息。以及通过使用量、使用时间及单价等信息，自动计算出的平台消费信息。该模块默认开启所有报表统计，同时也支持平台管理员对单个报表进行手动开启或关闭，开启/关闭后将在最长 20 分钟内，平台开始/停止采集报表数据，往期已采集到的数据还将正常展示。运营管理数据最多可在平台上保留 365 天，超过保留时间的统计数据将被自动删除。您也可以通过 CSV 或 Excel 方式下载报表后进行进一步的统计和分析。
```
Comment (windsonsea) [inline docs/zh/docs/ghippo/user-guide/report-billing/index.md:8]:
```suggestion
运营管理通过可视化的方式，为您展示平台上统计时间范围内集群、节点、命名空间、容器组、工作空间等维度的 CPU/内存/存储/GPU 的使用总量和使用率等信息。以及通过使用量、使用时间及单价等信息，自动计算出的平台消费信息。该模块默认开启所有报表统计，同时也支持平台管理员对单个报表进行手动开启或关闭，开启/关闭后将在最长 20 分钟内，平台开始/停止采集报表数据，往期已采集到的数据还将正常展示。运营管理数据最多可在平台上保留 365 天，超过保留时间的统计数据将被自动删除。您也可以通过 CSV 或 Excel 方式下载报表后进行进一步的统计和分析。
```

File: docs/zh/docs/ghippo/user-guide/report-billing/index.md [hunk 2]
```diff
@@ -5,16 +5,16 @@ hide:
...
 
 运营管理仅对标准版及以上版本开放，社区版暂不支持。
 
 你需要先[安装或升级运营管理模块](./gmagpie-offline-install.md)，然后就可以体验报表管理和计费计量。
 
 ## 报表管理
 
-报表管理通过 CPU 使用量、 CPU 使用率、内存使用量、内存使用率、存储使用量、存储使用率 6 个维度，对集群、节点、容器组、工作空间、命名空间 5 种资源进行数据统计。同时联动审计和告警模块，支持对审计数据和告警数据进行统计管理。共计支持 7 种类型报表。
+报表管理通过 CPU 利用率、内存利用率、存储利用率、GPU算力利用率、GPU显存利用率 5 个维度，对集群、节点、容器组、工作空间、命名空间 5 种资源进行数据统计。同时联动审计和告警模块，支持对审计数据和告警数据进行统计管理。共计支持 7 种类型报表。
```
Comment (windsonsea) [inline docs/zh/docs/ghippo/user-guide/report-billing/index.md:16]:
```suggestion
报表管理通过 CPU 利用率、内存利用率、存储利用率、GPU 算力利用率、GPU 显存利用率 5 个维度，对集群、节点、容器组、工作空间、命名空间 5 种资源进行数据统计。同时联动审计和告警模块，支持对审计数据和告警数据进行统计管理。共计支持 7 种类型报表。
```

File: docs/zh/docs/ghippo/user-guide/report-billing/index.md [hunk 3]
```diff
@@ -5,16 +5,16 @@ hide:
...
 
-## 计费计量
+## 计量计费
 
-计量计费针对平台上的集群、节点、容器组、命名空间和工作空间 5 种资源进行计费统计。根据不同资源中 CPU 、内存和存储的使用量，以及用户手动配置的价格和货币单位自动计算出每种资源在统计时间的消费情况，根据所选时间跨度不同，可快速计算出该跨度内的实际消费情况，如月度、季度、年度等。
+计量计费针对平台上的集群、节点、容器组、命名空间和工作空间 5 种资源进行计费统计。根据不同资源中 CPU 、内存、存储和 GPU 的使用量，以及用户手动配置的价格和货币单位自动计算出每种资源在统计时间的消费情况，根据所选时间跨度不同，可快速计算出该跨度内的实际消费情况，如月度、季度、年度等。
```
Comment (windsonsea) [inline docs/zh/docs/ghippo/user-guide/report-billing/index.md:20]:
```suggestion
计量计费针对平台上的集群、节点、容器组、命名空间和工作空间 5 种资源进行计费统计。根据不同资源中 CPU、内存、存储和 GPU 的使用量，以及用户手动配置的价格和货币单位自动计算出每种资源在统计时间的消费情况，根据所选时间跨度不同，可快速计算出该跨度内的实际消费情况，如月度、季度、年度等。
```

**Review by windsonsea — APPROVED** [review]
nits

**Pull request merged**